src/api/nodeRpcCall.js

```diff
@@ -1845,14 +1834,19 @@ const congressProposeLegislationReferendum = async (
     referendumPropose;
 
   const threshold = await congressMajorityThreshold();
+  const existingPreimages = await api.query.preimage.preimageFor([addLaw.hash, addLaw.encodedLength])
```

Comment:
```suggestion
  const existingPreimage = await api.query.preimage.preimageFor([addLaw.hash, addLaw.encodedLength])
```

But TBH I would even reduce it to just `preimage`.